nhais/Pipfile

```diff
@@ -12,6 +12,7 @@ tornado = "*"
 integration-adaptors-common = {editable = true,path = "./../common"}
 pycurl = {version = "~=7.43",platform_system = "!= 'Windows'"}
 marshmallow = "~=3.2"
+aioboto3 = "*"
```

Comment:
We should use a concrete version here. 8.0.3 seems to be the latest

Reply:
ok